### PR TITLE
doc/dev: use http://docs.ceph.com/en/latest/ instead of /docs/master/ for docs

### PR DESCRIPTION
**Pull request by Rethan**
Several links under http://docs.ceph.com/docs/master/ were unable to access.
Change them to http://docs.ceph.com/en/lastest so we can access them directly.

Signed-off-by: haoyixing <haoyixing@kuaishou.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by Rethan** [comment]
Maybe the commit title is not quite proper since some code comments were also changed. But it didn't come to my mind how to write the title. A proper title suggestion is welcomed.

**Comment by Rethan** [comment]
Besides, url of http://docs.ceph.com/docs/master/rados/api/librados/#asychronous-io under examples/librados was not accessible either, but i didn't find a alternertive after searching ceph docs.

**Comment by zdover23** [comment]
At a minimum, you'll need to rename this pull request. I suggest that you change the title to "doc/dev ...", where the "..." is the rest of the title. That'll fix these build errors that you're getting (probably), and then we can assess the rest of the PR.

git commit --amend

or you can rebase and squash, if that doesn't work for you.

**Comment by Rethan** [comment]
jenkins test docs

**Comment by Rethan** [comment]
> At a minimum, you'll need to rename this pull request. I suggest that you change the title to "doc/dev ...", where the "..." is the rest of the title. That'll fix these build errors that you're getting (probably), and then we can assess the rest of the PR.
> 
> git commit --amend
> 
> or you can rebase and squash, if that doesn't work for you.

still causing doc build errors and i have no clue...

File: doc/rados/operations/pools.rst [hunk 1]
```diff
@@ -276,21 +276,21 @@ You may set values for the following keys:
 
 ``compression_algorithm``
 
-:Description: Sets inline compression algorithm to use for underlying BlueStore. This setting overrides the `global setting <http://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#inline-compression>`_ of ``bluestore compression algorithm``.
+:Description: Sets inline compression algorithm to use for underlying BlueStore. This setting overrides the `global setting <https://docs.ceph.com/en/latest/rados/configuration/bluestore-config-ref/#inline-compression>`_ of ``bluestore compression algorithm``.
```
Comment (tchaikov) [inline doc/rados/operations/pools.rst:279]:
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/rados/operations/pools.rst:3: WARNING: Duplicate explicit target name: "global setting".
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/rados/operations/pools.rst:3: WARNING: Duplicate explicit target name: "global setting".
```

see https://jenkins.ceph.com/job/ceph-pr-docs/36648/console

there are two options:

- unnamed link, as suggested by https://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text
- add a named hyper link: see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

**Comment by zdover23** [comment]
jenkins render docs

**Comment by zdover23** [comment]
@Rethan Change "lastest" to "latest" in the title of this PR and the commit message, and I'll check it again and merge it.

**Comment by Rethan** [comment]
> @Rethan Change "lastest" to "latest" in the title of this PR and the commit message, and I'll check it again and merge it.

Done, thanks for pointing out.

**Comment by zdover23** [comment]
jenkins retest this please

**Comment by Rethan** [comment]
jenkins test dashboard

**Comment by Rethan** [comment]
> jenkins retest this please

Seems working now.

**Comment by zdover23** [comment]
I'm still waiting on a few checks to complete.

**Comment by Rethan** [comment]
> I'm still waiting on a few checks to complete.

It seems that after quoting your jenkins command, it just does the test again....I'm sorry if that disturbs you, now it looks ok.

**Comment by zdover23** [comment]
> > I'm still waiting on a few checks to complete.
> 
> It seems that after quoting your jenkins command, it just does the test again....I'm sorry if that disturbs you, now it looks ok.

I'm not disturbed. I was just initiating all the tests again because four of them were failing before. I suspect that the failing of those tests has more to do with the Jenkins infrastructure than it does with your commit. If Kefu says that this change meets with his approval, it'll be merged.

**Comment by Rethan** [comment]
@zdover23 ping? Wondering if there any remaining problems.

File: doc/dev/freebsd.rst [hunk 2]
```diff
@@ -44,7 +44,7 @@ MON creation
 
 Monitors are created by following the manual creation steps on::
 
-  http://docs.ceph.com/docs/master/install/manual-deployment/
+  https://docs.ceph.com/en/latest/install/manual-freebsd-deployment/
```
Comment (tchaikov) [inline doc/dev/freebsd.rst:47]: @wjwithagen could you help review this change?

**Review by tchaikov — APPROVED** [review]
LGTM. needs the blessings from @wjwithagen

**Review by callithea — APPROVED** [review]
lgtm

**Review by wjwithagen — APPROVED** [review]
This look oke.

**Pull request merged**